BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -281,15 +281,15 @@ informative:
 
 An earlier version of this document was published as RFC 7525 when the industry was in the midst of its transition to TLS 1.2. Years later this transition is largely complete and TLS 1.3 is widely available. This document updates the guidance given the new environment and obsoletes RFC 7525. In addition, the document updates RFC 5288 and RFC 6066 in view of recent attacks.
 
 --- middle
 
 # Introduction
 
-Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS) are widely used to protect data exchanged over application protocols such as HTTP {{HTTP1.1}} {{HTTP2}}, SMTP {{?RFC5321}}, IMAP {{?RFC9051}}, POP {{?STD53}}, SIP {{?RFC3261}}, and XMPP {{?RFC6120}}.  Over the years leading to 2015, the industry had witnessed serious attacks on TLS and DTLS, including attacks on the most commonly used cipher suites and their modes of operation.  For instance, both the AES-CBC {{?RFC3602}} and RC4 {{!RFC7465}} encryption algorithms, which together were once the most widely deployed ciphers, were attacked in the context of TLS.  Detailed information about the attacks known prior to 2015 is provided in a companion document ({{?RFC7457}}) to the previous version of this specification, which will help the reader understand the rationale behind the recommendations provided here. That document has not been updated in concert with this one; instead, newer attacks are described in this document, as are mitigations for those attacks.
+Transport Layer Security (TLS) is used to protect data exchanged over a wide variety of application protocols, including HTTP {{HTTP1.1}} {{HTTP2}}, IMAP {{?RFC9051}}, POP {{?STD53}}, SIP {{?RFC3261}}, SMTP {{?RFC5321}}, and XMPP {{?RFC6120}}.  Those protocols use both the TLS handshake protocol and the TLS record layer.  The TLS handshake protocol is also used with different record layers in transport protocols such as Datagram Transport Layer Security (DTLS) and QUIC {{?RFC9000}}.  Over the years leading to 2015, the industry had witnessed serious attacks on the TLS "family" of protocols, including attacks on the most commonly used cipher suites and their modes of operation.  For instance, both the AES-CBC {{?RFC3602}} and RC4 {{!RFC7465}} encryption algorithms, which together were once the most widely deployed ciphers, were attacked in the context of TLS.  Detailed information about the attacks known prior to 2015 is provided in a companion document ({{?RFC7457}}) to the previous version of this specification, which will help the reader understand the rationale behind the recommendations provided here. That document has not been updated in concert with this one; instead, newer attacks are described in this document, as are mitigations for those attacks.
```

Comment:
It's not entirely accurate to characterise DTLS as reusing the TLS handshake to apply the result to a different record layer. DTLS is armed to address the lack of an underlying reliable pipe abstraction (i.e., amplification potential, retransmission, reordering, path migration, etc.) so even if the bulk of the handshake is the same, there are quite important differences in terms of state machine and messages exchanged.

Reply:
Hmm, true. I will propose further changes.

Reply:
QUIC on the other hand behaves exactly as you phrase it: the QUIC transport recreates the TCP service and feeds a reliable, ordered stream into the TLS machinery.

Reply:
See https://github.com/yaronf/I-D/pull/390/commits/23824dc685c44103b9b513b5a10f9916c68598ac